lib/_http_server.js

```diff
@@ -225,6 +226,30 @@ function ServerResponse(req) {
 ObjectSetPrototypeOf(ServerResponse.prototype, OutgoingMessage.prototype);
 ObjectSetPrototypeOf(ServerResponse, OutgoingMessage);
 
+ServerResponse.prototype.sendFile = function(file, options, callback) {
+  if (typeof options === 'function') {
+    callback = options;
+    options = {};
+  }
+
+  validateObject(options, 'options');
+  validateCallback(callback, 'callback');
+
+  const { start, end, highWaterMark } = options;
+
+  let path
+  let fd
+
+  if (file instanceof FileHandle || Number.isInteger(file)) {
+    fd = file
+  } else {
+    path = file
+  }
+
+  // TODO (perf): Optimize this...
+  pipeline(fs.createReadStream(path, { fd, start, end, highWaterMark }), callback);
```

Comment:
> read buffers through the fs api and write to the response

Is there any similar implementation or which apis to use could you guide? would like to give it a try!